### PR TITLE
twitter btn addition

### PR DESCRIPTION
**Pull request by prahaladbelavadi**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -178,7 +198,7 @@ <h2>Security</h2>
   			      return $.get('https://api.blockcypher.com/v1/btc/main/addrs/3LPbF1wufpLzRU5pBNutivadVXyK9jwtkP/balance')
                             }).then((d) => {
 			      var balance = parseFloat(d.balance / 1e8).toFixed(4)
-			      balanceElement.textContent = balance + '/' + parseFloat((350000 / price)).toFixed(2) + ' BTC Raised'; 
+			      balanceElement.textContent = balance + '/' + parseFloat((350000 / price)).toFixed(2) + ' BTC Raised';
```
Comment (masonicGIT) [inline index.html:201]: Can you reset this so the only changes are for the twitter button

File: index.html [hunk 2]
```diff
@@ -10,6 +10,7 @@
 		<meta charset="utf-8" />
 		<meta name="viewport" content="width=device-width, initial-scale=1" />
 		<link rel="stylesheet" href="assets/css/main.css" />
+		<link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
```
Comment (masonicGIT) [inline index.html:13]: What is the purpose of this? Is it for the twitter css class?

File: index.html [hunk 3]
```diff
@@ -69,17 +70,36 @@ <h2>Participate</h2>
...
+								</a>
+
+							</div>
 						</div>
 						<div class="image">
 						  <a href="https://insight.bitpay.com/address/3LPbF1wufpLzRU5pBNutivadVXyK9jwtkP" target="blank">
 							<img src="images/3LPbF1wufpLzRU5pBNutivadVXyK9jwtkP.png" alt="" />
 						  </a>
 						</div>
+
```
Comment (masonicGIT) [inline index.html:96]: Try to remove these so that the only lines that are changed, are the ones that we intend to change

**Review by masonicGIT — COMMENTED** [review]
Just a few comments.

File: index.html (hunk 2 above)
Comment (prahaladbelavadi) [reply index.html:13]: Font awesome is css is for the twitter glyphicon in the button.
Twitter Icon makes the button look visually appealing

**Comment by prahaladbelavadi** [comment]
I'll remove the whitespace and send in another pull request at https://github.com/masonicGIT/btcwizard/pull/8

**Pull request closed without merging**